MAUI/Picker/getting-started.md

```diff
@@ -0,0 +1,308 @@
+
+## Creating an application using the .NET MAUI Picker
+
+1. Create a new .NET MAUI application in Visual Studio.
+
+2. Syncfusion .NET MAUI components are available on [nuget.org](https://www.nuget.org/). To add `SfPicker` to your project, open the NuGet package manager in Visual Studio, search for [Syncfusion.Maui.Picker](https://www.nuget.org/packages/Syncfusion.Maui.Picker), and then install it.
+
+3. To initialize the control, import the control namespace `Syncfusion.Maui.Picker` in XAML or C# code.
+
+4. Initialize `SfPicker.`
+
+{% tabs %}
+{% highlight xaml tabtitle="XAML" hl_lines="3 5" %}
+
+<ContentPage   
+    . . .
+    xmlns:picker="clr-namespace:Syncfusion.Maui.Picker;assembly=Syncfusion.Maui.Picker">
+
+    <picker:SfPicker />
+</ContentPage>
+
+{% endhighlight %}
+{% highlight c# tabtitle="C#" hl_lines="1 9 10" %}
+
+using Syncfusion.Maui.Picker;
+. . .
+
+public partial class MainPage : ContentPage
+{
+    public MainPage()
+    {
+        InitializeComponent();
+        SfPicker picker = new SfPicker();
+        this.Content = picker;
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+## Register the handler
+
+The `Syncfusion.Maui.Core` NuGet is a dependent package for all Syncfusion controls of .NET MAUI. In the `MauiProgram.cs` file, register the handler for Syncfusion core.
+
+{% tabs %}
+{% highlight C# tabtitle="MauiProgram.cs" hl_lines="1 10" %}
+
+using Syncfusion.Maui.Core.Hosting;
+namespace GettingStarted
+{
+    public static class MauiProgram
+    {
+        public static MauiApp CreateMauiApp()
+        {
+            var builder = MauiApp.CreateBuilder();
+
+            builder.ConfigureSyncfusionCore();
+            builder
+            .UseMauiApp<App>()
+            .ConfigureFonts(fonts =>
+            {
+                fonts.AddFont("OpenSans-Regular.ttf", "OpenSansRegular");
+                fonts.AddFont("Segoe-mdl2.ttf", "SegoeMDL2");
+            });
+
+            return builder.Build();
+        }
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+## Set header to the Picker
+
+The SfPicker control allows you to add the header text by setting the [Text] property in the [PickerHeaderView]. To enable the header view by setting the [Height] property in [PickerHeaderView].
+
+{% tabs %}
+{% highlight xaml tabtitle="XAML" hl_lines="3" %}
+
+<picker:SfPicker x:Name="picker">
+    <picker:SfPicker.HeaderView>
+        <picker:PickerHeaderView Text="Select a color" Height="40" />
+    </picker:SfPicker.HeaderView>
+</picker:SfPicker>
+
+{% endhighlight %}
+{% highlight c# tabtitle="C#" hl_lines="4 5" %}
+
+SfPicker picker = new SfPicker();
+picker.HeaderView = new PickerHeaderView()
+{
+    Text = "Select a color",
+    Height = 40,
+};
+
+this.Content = picker;
+
+{% endhighlight %}
+{% endtabs %}
+
+## Add picker items to the picker
+
+The .NET MAUI Picker control is a data bounded control. Hence, you must create collection of data’s and bind it to the picker control.
```

Comment:
The .NET MAUI Picker control is a data-bound control. Hence, you must create a data collection and bind it to the picker control.

Reply:
Content modified.